var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -75,6 +76,7 @@ class Tau(Package):
     variant('ppc64le', default=False, description='Build for IBM Power LE nodes')
     variant('x86_64', default=False, description='Force build for x86 Linux instead of auto-detect')
 
+    depends_on('zlib', type='build')
```

Comment:
```suggestion
    depends_on('zlib', type='link')
```
Zlib has no binaries to add to the `PATH`, only libraries and header files to link to.